packages/blast-tests/e2e-tests/tests/init-d.test.sh

```diff
@@ -4,11 +4,11 @@ source ./packages/blast-tests/e2e-tests/vars.sh
 echo -n 'blast init -d...'
 blast init -d $INIT_FOLDER &> /dev/null && cd $INIT_FOLDER
 
-if [[ ! `ls` == $TEMPLATE_FILES || ! `ls scripts` == $TEMPLATE_SCRIPTS_FILES ]]; then
+if [[ `ls` == $TEMPLATE_FILES || ! `ls scripts` == $TEMPLATE_SCRIPTS_FILES ]]; then
```

Comment:
To invert the logic it should be
if [[ `ls` == $TEMPLATE_FILES && `ls scripts` == $TEMPLATE_SCRIPTS_FILES ]]